src/controllers/signController.ts

```diff
@@ -70,10 +70,8 @@ export class SignController {
 
             return new SignTransactionResponse(toBase64(signedTransaction));
         } else {
-            // for simulated transactions we use timestamp as tx ID
-            return new SignTransactionResponse(toBase64({
-                txId: Date.now().toFixed()
-            }));
+            // for simulated transactions we will use operation ID as tx ID
+            return new SignTransactionResponse(toBase64({}));
```

Comment:
No argument passed. Is it correct?

Reply:
Yes, it is. It's just a simple way to say that operation ID should be used. With minimal change of existent code and logic on both sides (sign-service and api-service).
Actually, there is an argument - empty object )